lms/djangoapps/ccx/views.py

```diff
@@ -422,20 +446,17 @@ def ccx_invite(request, course, ccx=None):
     identifiers = _split_input_list(identifiers_raw)
     auto_enroll = True
     email_students = True if 'email-students' in request.POST else False
-    for identifier in identifiers:
-        user = None
-        email = None
-        try:
-            user = get_student_from_identifier(identifier)
-        except User.DoesNotExist:
-            email = identifier
-        else:
-            email = user.email
-        try:
-            validate_email(email)
-            course_key = CCXLocator.from_course_locator(course.id, ccx.id)
-            email_params = get_email_params(course, auto_enroll, course_key=course_key, display_name=ccx.display_name)
-            if action == 'Enroll':
+    course_key = CCXLocator.from_course_locator(course.id, ccx.id)
+    ccx_course_overview = CourseOverview.get_from_id(course_key)
+    email_params = get_email_params(course, auto_enroll, course_key=course_key, display_name=ccx.display_name)
+
+    if action == 'Enroll':
+        for identifier in identifiers:
```

Comment:
i am saying you can create 2 methods and move enrolment and unenrolment code their.
because we are doing this in two methods ```ccx_invite and ccx_student_management```.
You can create new methods mention above which will handle enrolment and unenrolment for single and multiple users.